### DISCUSSION TITLE
Can you please update the pypi package for torchgeo?

### DISCUSSION THREAD

**Discussion by metazool** [torchgeo/torchgeo#617] [General]
There has been some good work added, new datamodules, since the last release to pypi in March. It's worth publishing an update (so the scripts in the repo work with the packaged version)

https://pypi.org/project/torchgeo/

**Comment by ashnair1** [comment]
A 0.3.0 release is planned for sometime around July 4th. Ref: microsoft/torchgeo/discussions/602

**Reply by metazool** [reply]
that's great to hear! thank you

**Comment by adamjstewart** [comment]
v0.3.0 is now released: https://github.com/microsoft/torchgeo/releases/tag/v0.3.0
